tests/analytics/conftest.py

```diff
@@ -129,3 +132,9 @@ def datadog_client(mocker):
             side_effect=partial(api_usage.collect_analytics_data, client=client),
         )
     return client
+
+
+@pytest.fixture
+def sentry_respx_mock(respx_mock):
```

Comment:
_nit_ : Pour éviter le shadowing si jamais une autre _fixture_ du fichier venait à en avoir besoin :
```suggestion
@pytest.fixture(name="sentry_respx_mock_")
def sentry_respx_mock_fixture(respx_mock):
```